tools/monorepo/composer.json

```
@@ -1,5 +1,5 @@
 {
     "require": {
-        "contao/monorepo-tools": "dev-master"
+        "contao/monorepo-tools": "dev-master#608d7dee82a1dba95be989b0b843634e9a2da085"
```

Should get reverted after we reverted https://github.com/contao/monorepo-tools/commit/a4ff52a5b125c50e22d43b22ccdcb7a5a9801095
```suggestion
        "contao/monorepo-tools": "dev-master"
```